include/git2/attr.h

```diff
@@ -120,12 +120,20 @@ GIT_EXTERN(git_attr_value_t) git_attr_value(const char *attr);
 
 /**
  * Check attribute flags: Using the system attributes file.
+ * Check attribute flags: controlling extended attribute behavior.
```

Comment:
Was the intent here to erase the preceding line?

Reply:
Fixed.